### DISCUSSION TITLE
zero-contrib

### DISCUSSION THREAD

**Discussion by MutionHu** [suyuan32/simple-admin-core#189] [General]
建议对goctl的优化改动以zero-contrib的方式并入go-zero中，可借鉴https://github.com/zeromicro/zero-contrib

**Comment by suyuan32** [comment]
没办法并入，这个工具目前大部分功能和原版 goctl 已经不一样了，而且大部分功能都是针对 simple admin 的，并不是通用的

**Reply by MutionHu** [reply]
了解

**Comment by suyuan32** [comment]
go zero 官方有在关注本项目的，一些比较好的功能官方已经采纳并添加了，会有用户向官方反馈的，好的功能官方会添加的，有几个功能已经在官方有提供了
